apps/coscrad-frontend/src/components/login-button/login-button.tsx

```diff
@@ -1,13 +1,14 @@
 import { useAuth0 } from '@auth0/auth0-react';
-import { Button } from '@mui/material';
+import LoginIcon from '@mui/icons-material/Login';
+import { IconButton } from '@mui/material';
 
 const LoginButton = () => {
     const { loginWithRedirect } = useAuth0();
 
     return (
-        <Button variant="contained" onClick={() => loginWithRedirect()}>
-            Log In
-        </Button>
+        <IconButton color="primary" onClick={() => loginWithRedirect()}>
+            <LoginIcon />
```

Comment:
At some point, we'll need to make sure that this doesn't break the tests' handle on the button in Cypress.

Reply:
Right.  This must be the case for a bunch of MUI components.  Maybe we have to look for what HTML element React renders.

Reply:
Could we use classnames to embed test id props for testing?

<img width="351" alt="image" src="https://user-images.githubusercontent.com/63014783/225135332-29057e49-57ed-41af-a591-8443bac5f295.png">


<img width="245" alt="image" src="https://user-images.githubusercontent.com/63014783/225135243-595b3150-1cad-4eab-82f3-1a8f251760e7.png">

Reply:
I still don't understand what the holdup is on using data attributes for this. It's exactly what they are for.